slave.mk

```diff
@@ -262,6 +262,7 @@ $(info "TELEMETRY_WRITABLE"              : "$(TELEMETRY_WRITABLE)")
 $(info "PDDF_SUPPORT"                    : "$(PDDF_SUPPORT)")
 $(info "MULTIARCH_QEMU_ENVIRON"          : "$(MULTIARCH_QEMU_ENVIRON)")
 $(info "SONIC_VERSION_CONTROL_COMPONENTS": "$(SONIC_VERSION_CONTROL_COMPONENTS)")
+$(info "INSTALL_THIRD_PARTY_DEBS"        : "$(INSTALL_THIRD_PARTY_DEBS)")
```

Comment:
>INSTALL_THIRD_PARTY_DEBS [](http://example.com/codeflow?start=8&length=24)

This new parameter is too general and easy to abuse.

Reply:
Could you please explain? If so, is there an alternative to what we are trying to achive?